src/compas_viewer/scene/scene.py

```diff
@@ -70,10 +81,35 @@ def __init__(self, name: str = "ViewerScene", context: str = "Viewer"):
 
         #  Primitive
         self.objects: list[ViewerSceneObject]
```

Comment:
btw, this does not initialize the attribute. it just adds a type annotation. if you try to access this attribute an error will be thrown. it should be `self.objects: list[ViewerSceneObject] = []`

Reply:
`objects` is inherent from the parent class, I believe this was only for additionally type-hinting the object will be `ViewerScenceObject` instead of the base `SceneObject`